taxonomy-editor-frontend/src/pages/editentry/ListAllProperties.jsx

```diff
@@ -1,19 +1,30 @@
 import { IconButton, TextField, Typography } from "@mui/material";
 import AddBoxIcon from '@mui/icons-material/AddBox';
-import { DataGrid } from '@mui/x-data-grid';
+import MaterialTable from '@material-table/core';
+import { useState } from "react";
 
 const ListAllProperties = ({ nodeObject, setNodeObject }) => {
     let toBeRendered = []
+
+    function guidGenerator() {
+        var S4 = function() {
+           return (((1+Math.random())*0x10000)|0).toString(16).substring(1);
+        };
+        return (S4()+S4()+"-"+S4()+"-"+S4()+"-"+S4()+"-"+S4()+S4()+S4());
+    }
+
     Object.keys(nodeObject).forEach((key, index) => {
         if (key.startsWith('prop')) {
                 let property_name = key.split('_').slice(1).join('_');
```

Comment:
please include in a comment how the key looks like. Otherwise it is difficult to follow why the split.slice.join